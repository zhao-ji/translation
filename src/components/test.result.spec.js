import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Result from './result';

describe('<GoogleResult />', () => {
    describe('render', () => {
        test('default correctly', () => {
            const props = {
            };
            const component = shallow(<Result.GoogleResult {...props}/>);
            expect(toJson(component)).toMatchSnapshot();
        });
        test('render result', () => {
            const props = {
                title: "hello",
            };
            const component = shallow(<Result.GoogleResult {...props}/>);
            expect(toJson(component)).toMatchSnapshot();
        });
    });
});

describe('<BaiduResult />', () => {
    describe('render()', () => {
        test('default', () => {
            const props = {
            };
            const component = shallow(<Result.BaiduResult {...props}/>);
            expect(toJson(component)).toMatchSnapshot();
        });
        test('render result', () => {
            const props = {
                title: "hello",
            };
            const component = shallow(<Result.BaiduResult {...props}/>);
            expect(toJson(component)).toMatchSnapshot();
        });
    });
});
