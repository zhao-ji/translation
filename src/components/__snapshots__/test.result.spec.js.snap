// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaiduResult /> render() default 1`] = `
<TranslationCard
  header="Baidu"
/>
`;

exports[`<BaiduResult /> render() render result 1`] = `
<TranslationCard
  header="Baidu"
/>
`;

exports[`<GoogleResult /> render default correctly 1`] = `
<TranslationCard
  header="Google"
/>
`;

exports[`<GoogleResult /> render render result 1`] = `
<TranslationCard
  header="Google"
/>
`;
